var/spack/repos/builtin/packages/r-scater/package.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RScater(RPackage):
+    """Single-Cell Analysis Toolkit for Gene Expression Data in R.
+
+       A collection of tools for doing various analyses of single-cell RNA-seq
+       gene expression data, with a focus on quality control and
+       visualization."""
+
+    homepage = "https://bioconductor.org/packages/scater"
+    git      = "https://git.bioconductor.org/packages/scater.git"
+
+    version('1.12.2', commit='1518dc27a87c79181c34107d639e95fe55e55092')
+    version('1.10.1', commit='2e6694af2929092f263c2b0830d48b3f9632e70c')
+    version('1.8.4', commit='d560a9a378541d53d17990d2aa2cd28874df3dcd')
+    version('1.6.3', commit='964effb4e883102d7c8cae627dbac4ba5d216a75')
+    version('1.4.0', commit='90a2eab66ff82ba6dd7fbb33e41cd0ded20fa218')
+
+    depends_on('r@3.3:', type=('build', 'run'))
```

Comment:
The latest version doesn't have any specific R version requirement. Did it used to?

Reply:
Seems all the versions before bioconductor 3.9 did strangely, fixed